src/RTCDTMFSender.ts

```diff
@@ -0,0 +1,36 @@
+import { NativeModules, Platform } from "react-native";
+
+const { WebRTCModule } = NativeModules;
+const DEFAULT_DURATION = 300;
+const DEFAULT_INTER_TONE_GAP = 70;
+
+export default class RTCDTMFSender {
```

Comment:
I think it should be easy to implement toneBuffer (minus cancellation) by getting a callback from the native function when done.

Let's aim for the most complete API we can deliver.

Reply:
Android (org.webrtc) don't provide as such callback, it does provide method to get bufferTones https://chromium.googlesource.com/external/webrtc/+/HEAD/sdk/android/api/org/webrtc/DtmfSender.java

Reply:
👍